src/shared/proto/tbots_software_msgs.proto

```diff
@@ -22,5 +23,5 @@ message PrimitiveSetMsg
     TimestampMsg time_sent = 1;
 
     // Robot ID to RadioPrimitive map
-    map<uint32, PrimitiveMsg> robot_primitives = 2;
+    map<uint32, PrimitiveMsg> robot_primitives = 2 [(nanopb.fieldopt).max_count = 20];
```

Comment:
please add comment explaining where `20` comes from.

SSL-Vision can only support 16 IDs (0-15 inclusive) so that would be a good value here.

https://github.com/RoboCup-SSL/ssl-vision/blob/master/patterns/teams/standard2010_16.svg

Reply:
So didn't put the ssl vision bit there because this isn't really tied to vision, but added:
```
    // NOTE: The `max_count` for this field should be set to a number that is less then
    //       or equal to the maximum number of robots we expect to run
```